observability/logging/logging-6.0/.logging-enhancement-proposals.adoc

```diff
@@ -0,0 +1,1452 @@
+// Module included in the following assemblies:
```

Comment:
Do we typically put enhancement proposals in documentation?  At best I would think we would reference it in some way since it is subject to change and may/may not become quickly out of date

Reply:
Nope, and you'll notice its hidden file. I threw it in so that writers had it easily accessible in their editor. This PR is not for merging.